homeassistant/components/hassio/__init__.py

```diff
@@ -446,6 +463,17 @@ async def async_service_handler(service: ServiceCall) -> None:
             DOMAIN, service, async_service_handler, schema=settings.schema
         )
 
+    dev_reg = dr.async_get(hass)
+
+    async def async_mount_reload(service: ServiceCall) -> None:
+        """Handle service calls for Hass.io."""
+        if (device := dev_reg.async_get(service.data[ATTR_DEVICE_ID])) and device.name:
+            await supervisor_client.mounts.reload_mount(device.name)
```

Comment:
Does this call need error handling?

Reply:
Error handling was added. 
@frenck , can you please let me know if this is what you had in mind? Please note that error handling is also implemented [here](https://github.com/home-assistant-libs/python-supervisor-client/blob/03469b6ac1f73d408380bfff9636912986a77736/aiohasupervisor/client.py#L126-L159).
(I still need to add test coverage so keeping it as a draft for now.)